core/src/main/java/net/mabako/steamgifts/fragments/GiveawayListFragment.java

```diff
@@ -200,7 +200,8 @@ public void onEnterLeaveResult(String giveawayId, String what, Boolean success,
                     // we want to hide entered giveaways
                     adapter.removeGiveaway(giveawayId);
                 } else {
-                    adapter.notifyItemChanged(giveaway);
+                    // We refresh the entire dataset to let the other quick enter buttons know if the user still have enough points
+                    adapter.notifyDataSetChanged();
```

Comment:
What are the performance implications for this?  Would it make sense to only refresh all items if the setting is enabled?